Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationCircleStrokeSpin.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ActivityIndicatorAnimationCircleStrokeSpin.swift
+//  IBAnimatable
+//
+//  Created by phimage on 02/05/2018.
+//  Copyright © 2018 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+public class ActivityIndicatorAnimationCircleStrokeSpin: ActivityIndicatorAnimating {
+
+  // MARK: Properties
+
+  let beginTime: Double = 0.5
+  let strokeStartDuration: Double = 1.2
+  let strokeEndDuration: Double = 0.7
+
+  let timingFunction: CAMediaTimingFunction = CAMediaTimingFunction(controlPoints: 0.4, 0.0, 0.2, 1.0)
+
+  // MARK: ActivityIndicatorAnimating
+
+  public func configureAnimation(in layer: CALayer, size: CGSize, color: UIColor) {
+    let x = (layer.bounds.size.width - size.width) / 2
+    let y = (layer.bounds.size.height - size.height) / 2
+    let circle = ActivityIndicatorShape.stroke.makeLayer(size: size, color: color)
+    circle.frame = CGRect(x: x, y: y, width: size.width, height: size.height)
+    circle.add(defaultAnimation, forKey: "animation")
+    layer.addSublayer(circle)
+  }
+
+}
+
+// MARK: - Setup
+
+private extension ActivityIndicatorAnimationCircleStrokeSpin {
+
+  var defaultAnimation: CAAnimationGroup {
+    let groupAnimation = CAAnimationGroup()
+    groupAnimation.animations = [rotationAnimation, strokeEndAnimation, strokeStartAnimation]
+    groupAnimation.duration = strokeStartDuration + beginTime
+    groupAnimation.repeatCount = .infinity
+    groupAnimation.isRemovedOnCompletion = false
+    groupAnimation.fillMode = kCAFillModeForwards
+    return groupAnimation
+  }
+
+  var rotationAnimation: CABasicAnimation {
+    let rotationAnimation = CABasicAnimation(keyPath: "transform.rotation")
```

Comment:
I think we can use `CABasicAnimation(keyPath: .rotation)` now

Reply:
Yes, I forgot my enum ;(